### DISCUSSION TITLE
Bosch Twinguard 8750001213

### DISCUSSION THREAD

**Discussion by pietrekzad** [Koenkk/zigbee2mqtt#16565] [Devices]
Hi..
Sorry for my english.
I bought a used device:
https://www.zigbee2mqtt.io/devices/8750001213.html

The device is not detected in Z2M. I did reset the device but nothing changed. Only blinking orange LED and still not detected.
I use connbee II.
Do I need a BOSCH gate to work ? 

Thank for answer.

**Comment by Grandma-Betty** [comment]
@pietrekzad
I updated the Wiki a couple of days ago because it was lacking informations about the pairing process.
You should be able to pair it with my added instructions.
Be aware that I was only able to pair the device with the ZigBee coordinator (the device has to be close to the ZigBee coordinator when pairing it), but not with a ZigBee router. It sucks and I don't know why this happens, but that's the way it is currently.
Maybe @Koenkk knows a bit more about this issue.

**Comment by Sailing74** [comment]
I couldn't get the Conbee II to use the install code required for many current Bosch devices. I switched to a Sonoff stick. The install code works there with no issues so far.
